app/screens/AddEventScreen/AddEventScreen.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { View } from 'react-native'
+import styles from './AddEventScreen.styles'
+
+const {
+  containerStyle
+} = styles
+
+class AddEventScreen extends React.Component {
```

Comment:
import React, { Component } from 'react' and than use extends Component without react